packages/patternfly-4/content/design-guidelines/usage-and-behavior/card-view/design.md

```diff
@@ -0,0 +1,75 @@
+---
+path: "/design-guidelines/usage-and-behavior/card-view"
+---
+# Card view
+Card view organizes content into a grid of cards. It is useful for displaying a small to moderate amount of content in a way that is not visually overwhelming.
+![card-view--desktop](./img/card view - desktop.png)
```

Comment:
This image is not displaying in GiyHub preview. I think it's because you have spaces in the name.